project3/index.html

```diff
@@ -0,0 +1,113 @@
+                <path
+                    d="M31.54 65.965c-9.482 18.107-21.283 40.644-6.668 61.007a2 2 0 1 0 3.248-2.333c-13.177-18.36-2.01-39.684 6.962-56.82l.582-1.113C46.186 46.59 49.67 26.644 46.63 3.934a2 2 0 1 0-3.964.531c2.927 21.85-.423 41.039-10.543 60.388z" />
+                <path
+                    d="M21.544 65.965C12.06 84.072.26 106.609 14.876 126.972a2 2 0 1 0 3.248-2.333c-13.177-18.36-2.011-39.684 6.962-56.82l.582-1.113C36.19 46.59 39.674 26.644 36.632 3.934a2 2 0 1 0-3.963.531c2.927 21.85-.424 41.039-10.544 60.388zM110.33 128a3.985 3.985 0 0 0 2.828-1.171l5.67-5.673a4.001 4.001 0 0 0 0-5.657l-16.767-16.773c-.036-.036-.076-.064-.113-.098a3.96 3.96 0 0 0-.596-4.866l-2.835-2.836a3.953 3.953 0 0 0-1.67-.982 3.975 3.975 0 0 0-.278-.84l-6.21-13.101a4 4 0 0 0-.786-1.115A16.474 16.474 0 0 0 77.857 70a15.768 15.768 0 0 0-15.834 16.097 16.517 16.517 0 0 0 4.884 11.465 4 4 0 0 0 1.114.786l13.096 6.213a4 4 0 0 0 .845.29 3.955 3.955 0 0 0 .977 1.659l2.835 2.836a3.956 3.956 0 0 0 4.864.596c.034.037.062.078.098.113l16.767 16.774a3.985 3.985 0 0 0 2.828 1.171zm-27.5-27.054l-13.095-6.212a12.231 12.231 0 0 1-.27-17.287A11.796 11.796 0 0 1 77.857 74a12.565 12.565 0 0 1 8.888 3.717l6.21 13.1zm5.772 5.571l-2.835-2.836 9.923-9.926 2.835 2.836zm10.631-4.963l16.768 16.774-5.67 5.672-16.768-16.773z" />
+                <path
+                    d="M51.533 65.965c-9.483 18.107-21.284 40.644-6.668 61.006a2 2 0 1 0 3.248-2.332c-13.177-18.36-2.011-39.684 6.962-56.82l.582-1.113C66.179 46.59 69.663 26.644 66.598 4.002a1.987 1.987 0 0 0 .332-.008c.268.124 26.783 12.832 29.65 42.203a2 2 0 0 0 .396 1.012c1.704 2.254 1.704 2.254 1.507 6.875l-.005.135a2 2 0 0 0 .376 1.255c1.17 1.622 12.477 9.952 15.69 12.144a3.527 3.527 0 0 1 1.447 2.25 3.216 3.216 0 0 1-1.172 2.566c-.326.167-7.986 4.16-7.578 9.561a1.982 1.982 0 0 0 .162.65 6.592 6.592 0 0 0 3.814 3.535 2.763 2.763 0 0 1 .668 2.356 3.67 3.67 0 0 1-1.558 2.057 2 2 0 0 0-.13 3.115 4.111 4.111 0 0 1 1.267 1.773 3.217 3.217 0 0 1-.556 2.848 1.827 1.827 0 0 0-.108.184 4.883 4.883 0 0 1-1.66 1.638l2.864 2.865a8.45 8.45 0 0 0 2.284-2.548 7.187 7.187 0 0 0 .98-6.222 7.276 7.276 0 0 0-.962-1.918 6.845 6.845 0 0 0 1.474-2.89 6.742 6.742 0 0 0-1.677-6.026 3.995 3.995 0 0 0-1.534-.995 2.62 2.62 0 0 1-1.33-1.056c.236-1.796 3.241-4.26 5.407-5.37a4.034 4.034 0 0 0 .976-.705 7.027 7.027 0 0 0 2.349-5.803 7.425 7.425 0 0 0-3.191-5.181c-4.17-2.846-12.225-8.956-14.272-10.774.169-4.26.04-5.48-2.005-8.277C97.12 13.817 68.891.493 68.601.361a3.963 3.963 0 0 0-1.796-.354 3.95 3.95 0 0 0-4.147 4.458c2.926 21.85-.424 41.04-10.544 60.388zM82.176 118.824c-.963-.074-9.57-.543-15.508 5.81a2 2 0 1 0 2.92 2.732c4.472-4.784 10.959-4.633 12.157-4.562a64.785 64.785 0 0 0 17.26 1.18l-3.856-3.857a59.963 59.963 0 0 1-12.71-1.265 1.994 1.994 0 0 0-.263-.038z" />
+            </svg>
+            <h2>morning</h2>
+            <figure>
+                <img src="images/makeup.jpg" alt="lipgloss, concealer, and eyeshadow">
```

Comment:
Largest this image shows up is 132px wide in the browser, but this image is still 4160px wide -- so, two issues:
1. The images need to be resized and optimized for the browser.
1. For this design, the images -- both the photos and the SVGs -- are really downplayed. At the smaller, mobile size, the images aren't really understandable, including the SVG lineart. Sizing things via percentage only without any `min-width` or `max-width` often has this result, e.g. `min-width: 130px;` on the SVGs and `min-width: 260px;` on the photos would allow you to keep the adaptive sizing but stop things from being too tiny.